src/settingsView.ts

```diff
@@ -316,12 +319,34 @@ function htmlForWebview(vscode: vscodeTypes.VSCode, extensionUri: vscodeTypes.Ur
           div.appendChild(label);
           projectsElement.appendChild(div);
         }
+
+        const allEnabled = projects.every(p => p.enabled);
```

Comment:
You set this to a variable, then never use it, and redo the same calculation in the next line.

Reply:
oops!